help_to_heat/frontdoor/models.py

```diff
@@ -16,9 +16,3 @@ class Answer(utils.UUIDPrimaryKeyBase, utils.TimeStampedModel):
 
     class Meta:
         indexes = [models.Index(fields=["session_id"])]
-
-
-class Feedback(utils.UUIDPrimaryKeyBase, utils.TimeStampedModel):
-    session_id = models.UUIDField(editable=False, blank=True, null=True)
-    page_name = models.CharField(max_length=128, editable=False, blank=True, null=True)
-    data = models.JSONField(encoder=DjangoJSONEncoder, editable=False)
```

Comment:
I'm not convinced it is correct to move this. I think Feedback should probably still live within the frontdoor, since that is where it's being created. (FeedbackDownload can be in portal)

If we *are* moving it, then it looks like you've created a migration to make a Feedback table in portal, but it doesn't drop the existing Feedback table from frontdoor. But as I say, I don't think we should be moving it.

Reply:
If `Feedback` lives in frontdoor and `FeedbackDownload` lives in portal, I guess we still need to import `FeedbackDownload` from portal for it to be used in `Feedback`? Is this alright? :0